src/App.js

```diff
@@ -1,28 +1,37 @@
-import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Categories from './components/Categories';
+import Books from './components/Books';
+// import book from './components/bookComponent';
```

Comment:
### [OPTIONAL]

- [ ] It is always good practice to remove the commented code in your application. This ensures that your project is clean and professional. Remember you can always use git history to go back to the previous version of your project